src/actions.spec.ts

```diff
@@ -0,0 +1,16 @@
+import { isNgrxFormsAction } from './actions';
+
+describe('action', () => {
```

Comment:
You are testing a concrete function, so you should use its name for the `describe`. Also, don't write the name as a string but instead access the `name` property of the function.